docs/_plugins/nav-tabs.rb

```diff
@@ -0,0 +1,54 @@
+require "erb"
+require "securerandom"
+
+module Jekyll
+  module NavTabs
+    class NavTabsBlock < Liquid::Block
+      def render(context)
+        environment = context.environments.first
+        environment['navtabs'] = {} # reset each time
+        super
+
+        uuid = SecureRandom.uuid
+        template = ERB.new <<-EOF
+<ul class="nav nav-tabs" id="<%= uuid %>" role="tablist">
+<% environment['navtabs'].each_with_index do |(key, _), index| %>
+  <li class="nav-item">
+    <a <%= index == 0 ? 'class="nav-link active"' : 'class="nav-link"'%> id="<%= key %>-tab" data-toggle="tab" href="#<%= key %>" role="tab" aria-controls="<%= key %>" <%= index == 0 ? 'aria-selected="true"' : 'aria-selected="false"'%>><%= key %></a>
+  </li>
+<% end %>
+</ul>
+<div class="tab-content" id="<%= uuid %>-content">
```

Comment:
My only concern is that the content within this div does not have enough top padding. We could add another CSS rule to adjust it.

```css
.tab-content {
  padding-top: 0.2em;
}
```


<img width="542" alt="image" src="https://user-images.githubusercontent.com/3464508/79678983-50ed9580-81b6-11ea-8d84-0e96607544e1.png">

Reply:
Although on the bootstrap docs, it looks like they added the padding to the bottom of the nav, rather than the top of the content:

<img width="1473" alt="image" src="https://user-images.githubusercontent.com/3464508/79679011-b8a3e080-81b6-11ea-852c-bb800aa0093c.png">

Reply:
Adding the `.nav-tabs` class should actually add that margin to the bottom properly, but according to my local build, `.nav-tabs` doesn't do that

Reply:
uh i dont know much about what you're trying to suggest; what exactly do you want me to change if any?